extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -70,11 +70,13 @@ static public function isCorrectPageType() {
 		$wg = F::app()->wg;
 
 		$showableNameSpaces = array_merge( $wg->ContentNamespaces, [ NS_FILE, NS_BLOG_ARTICLE ] );
+		$isInShowableNamespace =
+			$wg->Title->exists() && in_array( $wg->Title->getNamespace(), $showableNameSpaces );
```

Comment:
`$wg->Title->inNamespaces($showableNamespaces)`?

Reply:
and why do you get title via $wg-Title and in other place from RequestContext? please unify